src/CalcManager/UnitConverter.cpp

```diff
@@ -922,6 +923,24 @@ void UnitConverter::TrimString(wstring& returnString)
     }
 }
 
+/// <summary>
+/// Get number of significant digits (integer part + fractional part) of a number</summary>
+/// <param name="value">the number</param>
+unsigned int UnitConverter::GetNumberSignificantDigits(std::wstring value)
+{
+    TrimString(value);
+    unsigned int numberSignificantDigits = static_cast<unsigned int>(value.size());
+    if (value.find(L'.') != value.npos)
```

Comment:
Same comment here and below.  My understanding is this should be `wstring::npos`